sql/src/main/java/io/crate/analyze/InsertFromSubQueryAnalyzedStatement.java

```diff
@@ -134,11 +146,20 @@ public InsertFromSubQueryAnalyzedStatement(AnalyzedRelation subQueryRelation,
                         symbol = InputColumns.create(generatedRef.generatedExpression(), sourceSymbols);
                     }
                 }
-                symbols.add(symbol);
+
                 if (SymbolVisitors.any(Symbols.IS_COLUMN, symbol)) {
-                    throw new IllegalArgumentException(String.format(Locale.ENGLISH,
-                        "Column \"%s\" is required but is missing from the insert statement", column.sqlFqn()));
+                    if (alwaysRequireColumn) {
```

Comment:
Would it make sense to utilize the `nullable` information for this instead?

Reply:
Currently, if `Reference` is a primary key, then the `nullable` still can be true. This commit feadcdc475b7996bf51f6f1a99e676a4b0c58490 changes this behaviour, but it also touches a bit information schema tables to adjust to the meta data changes.